geomstats/learning/ahc.py

```diff
@@ -0,0 +1,118 @@
+"""The Agglomerative Hierarchical Clustering (AHC) on manifolds."""
+
+from sklearn.cluster import AgglomerativeClustering
+
+import geomstats.backend as gs
+
+
+class AgglomerativeHierarchicalClustering(AgglomerativeClustering):
```

Comment:
Did you modify this class, w.r.t. to scikit-learn? Or are we "only" using it?

Reply:
I made it easier to use thanks to the distance parameter instead of the affinity parameter which in scikit-learn, when callable, should be a function that takes the data as input and return a distance matrix.
In my code, we just need to specify the distance and this function is automatically defined.
Plus I changed the default parameter of the linkage parameter for it to be compatible with non-euclidean metrics.